### PR TITLE
Upgrade ol v3.14.2

### PR DESCRIPTION
**Pull request by marcjansen**
Require v3.14.2 and update the `custom builds` example.

See also openlayers/ol3#5005

/cc @ahocevar

### CODE REVIEW

File: src/ol-custom.json [hunk 1]
```diff
@@ -28,21 +28,18 @@
...
       "ol.ENABLE_WEBGL=false",
       "ol.ENABLE_PROJ4JS=false",
-      "ol.ENABLE_IMAGE=false",
-      "goog.DEBUG=false"
+      "ol.ENABLE_IMAGE=false"
     ],
     "jscomp_error": [
       "*"
     ],
     "jscomp_off": [
```
Comment (ahocevar) [inline src/ol-custom.json]: What about @gberaudo's https://github.com/openlayers/ol3/issues/5005#issuecomment-194768745?

**Comment by ahocevar** [comment]
Looks good, I only have one concern about missing `analyzerChecks`.

File: src/ol-custom.json (hunk 1 above)
Comment (marcjansen) [inline src/ol-custom.json]: It threw an exception in my tests... Not 100% sure the workshop would need it here. Probably @gberaudo can chime in?

Comment (fredj) [inline src/ol-custom.json]: `analyzerChecks` is only available in recent version of the compiler, not the one we use in v3.14.2

Comment (gberaudo) [inline src/ol-custom.json]: I think v3.14.2 uses an older version of the compiler where `analyzerChecks` was not present.

Comment (ahocevar) [inline src/ol-custom.json]: Thanks @fredj and @gberaudo. So please merge @marcjansen.

**Comment by fredj** [comment]
We will need to add `analyzerChecks` later, see #47 

please merge

**Comment by marcjansen** [comment]
Thanks everybody!

**Pull request merged**